lib/screens/introspection_list_screen.dart

```diff
@@ -126,14 +127,30 @@ class IntrospectionListPage extends GetView<IntrospectionListScreenController> {
               currentDay: controller.selectedDate,
               firstDay: DateTime.utc(2025, 3, 20),
               lastDay: DateTime.now(),
+              eventLoader:
+                  (day) =>
+                      controller.notes
+                          .where(
+                            (note) =>
+                                note.date.year == day.year &&
+                                note.date.month == day.month &&
+                                note.date.day == day.day,
+                          )
+                          .toList(),
```

Comment:
同じ日付の内省を複数作成したら点がそれと同じ数日付の部分に4つまで表示された。
「内省があるかどうか」をユーザーに伝えるだけなら以下のようにして点を一つだけ表示するようにしても良いと思う。

```suggestion
              eventLoader: (day) {
                final events = controller.notes.where(
                  (note) =>
                      note.date.year == day.year &&
                      note.date.month == day.month &&
                      note.date.day == day.day,
                );
                return events.isNotEmpty ? [events.first] : [];
              },
```